tests/testthat/test-deployments.R

```diff
@@ -80,10 +88,15 @@ test_that("can read/write env vars", {
 
 test_that("can read env vars when none set", {
   app <- local_temp_app()
-  addTestDeployment(app, "test1", envVars = NA_character_)
+  path <- addTestDeployment(app, "test1", envVars = NULL)
+  deps <- deployments(app, excludeOrphaned = FALSE)
+  expect_equal(deps$envVars, list(character()))
+  expect_false("envVars" %in% rownames(read.dcf(path)))
```

Comment:
@aronatkins this is where we test what happens when we deliberately create a deployment without `envVars` — we test that `envVars` is not written to the DCF, but it's still successfully read. I'll add something similar for `version`.

Reply:
Thanks. I missed that this test was confirming the processing of an "old" DCF. Would you mind adding a comment / renaming the test to remind us that this is testing DCF compatibility upon upgrade?